nixos/modules/services/backup/borgbackup.nix

```
@@ -555,6 +553,19 @@ in {
             example = "--save-space";
           };
 
+          extraPackages = mkOption {
+            type = with types; listOf package;
+            default = [];
+            defaultText = literalExample "[];";
+            example = literalExample ''
+              with pkgs; [
+                btrfs-progs
+              ]
+            '';
+            description = ''
+              Extra packages to be added to the service script path. For example, this might be useful to create a BTRFS snapshot of your data in the <literal>$preHook</literal>.
```

```suggestion
              Packages to be added to the service script path. For example, this might be useful to create a BTRFS snapshot of your data in the <option>preHook</option>.
```
and some line breaks would be nice.

---

I'll add this.